backend/src/gamestate/game.rs

```diff
@@ -432,10 +432,9 @@ impl Game {
         played_card: Card,
         player_penalized: bool,
     ) -> Result<(), CreateStatusError> {
-        let new_player_name = self.get_current_player().unwrap().name();
         self.message_all(WSMsg::play_card(
             player_name.clone(),
-            new_player_name.clone(),
+            self.get_current_player().unwrap().name(),
```

Comment:
Prosím, ošetři unwrap (i když je aktuálně nemožné, aby nastal). Nemůžeme si dovolit, aby nám to při obhajobě spadlo, nahazování trvá příliš dlouho.

Reply:
Fixed.